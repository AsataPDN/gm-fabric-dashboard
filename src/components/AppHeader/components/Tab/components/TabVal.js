import styled from "styled-components";

const TabVal = styled.dd`
  margin: 0;
  display: inline-block;
`;

export default TabVal;
