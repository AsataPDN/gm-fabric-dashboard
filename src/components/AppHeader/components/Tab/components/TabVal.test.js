import React from "react";
import { shallow } from "enzyme";
import TabVal from "./TabVal";

describe("TabVal", () => {
  it("matches snapshot", () => {
    const aTabVal = shallow(<TabVal />);
    expect(aTabVal).toMatchSnapshot();
  });
});
