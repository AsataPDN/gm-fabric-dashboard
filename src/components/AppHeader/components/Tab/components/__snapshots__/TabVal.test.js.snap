// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabVal matches snapshot 1`] = `
.c0 {
  margin: 0;
  display: inline-block;
}

<dd
  className="c0"
/>
`;
